features/step_definitions/posts_steps.rb

```diff
@@ -27,6 +27,22 @@
   user.post(:status_message, :text => text, :public => true, :to => user.aspect_ids)
 end
 
+Given /^"([^"]*)" has a public post with text "([^"]*)" and poll "([^"]*)"$/ do |email, text, question|
+  user = User.find_by_email(email)
+  poll = Poll.new(question: question)
+  poll.poll_answers.build(answer: '1')
+  poll.poll_answers.build(answer: '2')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.